webui/react/src/hooks/useTheme.ts

```diff
@@ -51,13 +52,14 @@ const updateAntDesignTheme = (path: string) => {
  * and storybook Theme decorators and not individual components.
  */
 export const useTheme = (): ThemeHook => {
+  const currentTheme = (() => getIsDarkMode() ? DarkLight.Dark : DarkLight.Light);
   const [ branding, setBranding ] = useState(BrandingType.Determined);
-  const [ mode, setMode ] = useState(() => getIsDarkMode() ? DarkLight.Dark : DarkLight.Light);
-
+  const [ mode, setMode ] = useState(currentTheme);
```

Comment:
**suggestion:** we might need a separate type that's different from `DarkLight` because we want to support `Auto` (which mac provides). Maybe something like:
```
export enum Mode {
  Auto = 'auto',
  Dark = DarkLight.Dark,
  Light = DarkLight.Light,
}
```